src/forms/fields/Select/index.tsx

```diff
@@ -111,14 +115,20 @@ export const Select: React.FC<{
 
   const handleChange = useCallback(
     (incomingSelection: Option | Option[]) => {
+      let selectedOption
+      if (Array.isArray(incomingSelection)) {
+        selectedOption = incomingSelection
+      } else {
+        selectedOption = incomingSelection.value
```

Comment:
I'm confused by this line because it seems like `setValue` https://github.com/payloadcms/website/pull/103/files#diff-77d8c6c369fb3225c7be8c3ac652305d615300280f42ccf64452f389c8c04d3cR68 and `onChange` https://github.com/payloadcms/website/pull/103/files#diff-77d8c6c369fb3225c7be8c3ac652305d615300280f42ccf64452f389c8c04d3cR43 both take an `Option` type, but `incomingSelection.value` surely isn't an `Option`.